.eslintrc

```diff
@@ -13,6 +13,30 @@
     "jest/globals": true
   },
   "rules": {
+    "max-nested-callbacks": [
+      "error",
+      2
+    ],
+    "complexity": [
+      "error",
+      2
+    ],
+    "max-lines": [
+      "error",
+      700
+    ],
+    "max-lines-per-function": [
+    "error",
+    100
+    ],
+    "max-statements": [
+      "error",
+      50
+    ],
+    "max-depth": [
+      "error",
+      4
+    ],
```

Comment:
Just for the record, how many more errors were found with these rules?

Reply:
96 ignoring the errors on the test files